### PR TITLE
Added my repo

### PR DESCRIPTION
**Pull request by SathvigaaBharathi**

### CODE REVIEW

File: sathvigaa Bharathi-decentralised voting system
Comment (Snehsk) [inline sathvigaa Bharathi-decentralised voting system:1]: Please remove this file. @SathvigaaBharathi

Comment (SathvigaaBharathi) [reply sathvigaa Bharathi-decentralised voting system:1]: Which file should I remove?

Comment (Snehsk) [reply sathvigaa Bharathi-decentralised voting system:1]: sathvigaa Bharathi-decentralised voting system 
this file which you have uploaded earlier in which date is not in the rename of screenshot.

**Review by Snehsk — CHANGES_REQUESTED** [review]
Could you please close this and open new PR in which you have to upload png file which is rename by this : name-projectname-date 

And please upload only one image.

**Pull request closed without merging**